docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -66,6 +66,9 @@ abstract class Client {
     onSign: (message: string) => Promise<Cacao.Signature>
   }): Promise<string>;
 
+  // Unsubscribes from all subscription topics related to the most recently registered account, decouples the client from the registered account, and pauses watchSubscription requests.
```

Comment:
What does `decouples the client from the registered account` mean

Reply:
for instance service that is periodically triggering watchSubscription request is no longer aware of the account
probably depends of the implementation

Reply:
Shouldn't it also delete the actual; identity key

Reply:
The unregister is good when we want this device to forget about Notify API whatsoever. 

Clients are account agnostic. Why should we track most recently registered account? What is missing in he method is `account`. We had a phase in Chat API when we wanted to do similar thing. In the end we decided to fallback to always specifying the account for simplicity.


On the other hand,
I still think we should go mute/unmute route. Maybe not in this PR but in near future. Which would alternatively solve it.

Reply:
>Shouldn't it also delete the actual; identity key

why? user may want to use this account again later

Reply:
Then why would they unregister?